code/modules/projectiles/projectile/bullets/shotgun.dm

```diff
@@ -92,8 +97,9 @@
 		qdel(src)
 
 /obj/item/projectile/bullet/pellet/shotgun_improvised
```

Comment:
these arent really an issue, never used

Reply:
If it remains unnerfed then it'll become more used, it needs to stay at relative power levels since it's only improvised ammo

Reply:
i say leave these as is, as there is no reason to ever use it considering how autolathes work

Reply:
Nice improvised ammo nerf, when it needs to be buffed, you nerf... Good job zeskorion... No...